### ISSUE TITLE
It would be great to add a rake task to the gem for initializing

### ISSUE THREAD

**Issue by contentfree** [sporkd/compass-html5-boilerplate#6]
I always forget how my rails apps got the HTML5 boilerplate stuff initialized. I go 1) rake -T, 2) rails g and finally 3) google for rails gem html5 boilerplate, leading me here where I see the `compass init` command.

There ought to be a html5boilerplate:init rake task.

**Comment by sporkd** [comment]
I agree.  I actually suggested some time back to the compass user group that Compass itself should be using Thor to do installs/updates instead of the compass command.   Thor could still read from the manifest, but it would behave more like a generator, allowing you to choose which files you want to overwrite on on individual basis instead of using the --force option.

That would be the ideal solution, but I suppose in the meantime we could add a simple rake shortcut to the compass script.  I may try to get to this if someone does not beat me to it first.

**Comment by rajneesh-18** [comment]
What is the main problem in this issue?
